### PR TITLE
Use getByteTimeDomainData to support iOS Safari in meeting demo

### PR DESCRIPTION
**Pull request by aburnson**
*Issue #:* #287 

*Description of changes*

Use getByteTimeDomainData to support iOS Safari in meeting demo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 1]
```diff
@@ -962,15 +962,15 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
       document.getElementById('audio-preview').parentElement.style.visibility = 'hidden';
```
Comment (ltrung) [inline demos/browser/app/meetingV2/meetingV2.ts:962]: Do we need to update this to getByteTimeDomainData too? Also does this check work?

File: demos/browser/app/meetingV2/meetingV2.ts [hunk 2]
```diff
@@ -962,15 +962,15 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
...
     let frameIndex = 0;
     this.analyserNodeCallback = () => {
       if (frameIndex === 0) {
-        analyserNode.getFloatTimeDomainData(data);
+        analyserNode.getByteTimeDomainData(data);
         const lowest = 0.01;
         let max = lowest;
         for (const f of data) {
-          max = Math.max(max, Math.abs(f));
+          max = Math.max(max, (f - 128) / 128);
```
Comment (ltrung) [inline demos/browser/app/meetingV2/meetingV2.ts:973]: Could you explain more about this logic?

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 1 above)
Comment (aburnson) [reply demos/browser/app/meetingV2/meetingV2.ts:962]: Good call. Thanks.

File: demos/browser/app/meetingV2/meetingV2.ts (hunk 2 above)
Comment (aburnson) [reply demos/browser/app/meetingV2/meetingV2.ts:973]: This math normalizes the values. Byte time domain data is stored 0-255 where 0...128...256 would be equivalent to -1...0...1.

**Pull request merged**